src/main/resources/Help/Code/ExampleMacro.xml

```diff
@@ -143,7 +143,7 @@
 .macroexample {
   margin-top: 1em;
 }
-.macroexample > .tab-content {
+.macroexample &gt; .tab-content {
```

Comment:
This is CSS, should not be escaped.

Reply:
This is content from `xml` file and it wasn't done manually. It was the result of XWiki page export + `mvn xar:format`.
Some other examples:
 https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-sharepage/xwiki-platform-sharepage-ui/src/main/resources/XWiki/SharePage.xml#L694
and
https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/PanelsCode/NavigationConfigurationSheet.xml#L681.
I would be curious what were the followed steps to achieve https://github.com/xwiki-contrib/application-help-center/commit/8e23b48660a4b694003869e3035038d7f945e01c#diff-ce2ea75e374e460fb9c4cf50feaec826R146.